packages/client/components/ActionMeeting.tsx

```diff
@@ -72,6 +72,7 @@ const ActionMeeting = (props: Props) => {
         />
       </ResponsiveDashSidebar>
       <MeetingArea>
+        <Suspense fallback={''}>
```

Comment:
Just out of curiosity, is suspense actually working with Relay? 😅  if yes, do we want a nicer fallback like progress?

Reply:
Yep, the phase components are being lazily loaded which meant that the height calculation was initially incorrect. We're using the empty string fallback option in quite a few components - I think a skeleton UI fallback could work well but I'd prefer to get this PR merged and refactor that in a different issue